### PR TITLE
Restructure the "Why Ballerina" Section

### PR DESCRIPTION
**Pull request by praneesha**
## Purpose
Restructure the "Why Ballerina" section.

> Fixes #

## Check List

- [ ] **Page Addition**
  - [ ] Add `permalink` to pages
  - [ ] If contains empty folder(s), Add front-matter `redirect_to:`

- [ ] **Page Rename**
  - [ ] Add front-matter `redirect_from`
  - [ ] Add front-matter `redirect_to:` (If applicable)

### CODE REVIEW

File: _data/whyballerinanavswanlake.yml [hunk 1]
```diff
@@ -0,0 +1,42 @@
...
+
+- title: Concurrent
+  url: /learn/why-ballerina/concurrent/ 
+  active: concurrent
+
+- title: Reliable, Maintainable
+  url: /learn/why-ballerina/reliable-maintainable/ 
+  active: reliable-maintainable
+
+
```
Comment (keizer619) [inline _data/whyballerinanavswanlake.yml:26]: @praneesha Shall we removed these empty lines?

File: _data/learnnavswanlake.yml [hunk 2]
```diff
@@ -3,27 +3,27 @@
...
-    - title: Graphical
-      url: /learn/user-guide/why-ballerina/graphical/
-      active: graphical
-    - title: Concurrent
-      url: /learn/user-guide/why-ballerina/concurrent/
-      active: concurrent
-    - title: Reliable, Maintainable
-      url: /learn/user-guide/why-ballerina/reliable-maintainable/
-      active: reliable-maintainable
+# - title: Why Ballerina
```
Comment (keizer619) [inline _data/learnnavswanlake.yml:6]: @praneesha are these temporary deleted ?

File: _layouts/ballerina-why-ballerina-left-nav-pages-swanlake.html [hunk 3]
```diff
@@ -0,0 +1,98 @@
...
+
+
+
+     <script src="https://code.jquery.com/jquery-1.11.0.min.js"></script>
+     <script src="/js/abixTreeList.min.js"></script>
+     <script>
+         $(document).ready(function() {
+             $('#tree').abixTreeList();
+         });
+
```
Comment (keizer619) [inline _layouts/ballerina-why-ballerina-left-nav-pages-swanlake.html:72]: @praneesha Lets remove unnecessary empty lines

File: _layouts/ballerina-why-ballerina-left-nav-pages-swanlake.html [hunk 4]
```diff
@@ -0,0 +1,98 @@
...
+        var bodyclass = document.body;
+
+        if ($(this).scrollTop() > 200) {
+            $('.cTopLink').removeClass('dispalaynone');
+          
+        } else {
+            $('.cTopLink').addClass('dispalaynone');
+          
+        }
+
```
Comment (keizer619) [inline _layouts/ballerina-why-ballerina-left-nav-pages-swanlake.html:87]: @praneesha These unnecessary empty lines can be removed?

File: _data/learnnavswanlake.yml (hunk 2 above)
Comment (praneesha) [reply _data/learnnavswanlake.yml:6]: Removed.

File: _data/whyballerinanavswanlake.yml (hunk 1 above)
Comment (praneesha) [reply _data/whyballerinanavswanlake.yml:26]: Fixed.

File: _layouts/ballerina-why-ballerina-left-nav-pages-swanlake.html (hunk 4 above)
Comment (praneesha) [reply _layouts/ballerina-why-ballerina-left-nav-pages-swanlake.html:87]: Fixed.

File: _layouts/ballerina-why-ballerina-left-nav-pages-swanlake.html (hunk 3 above)
Comment (praneesha) [reply _layouts/ballerina-why-ballerina-left-nav-pages-swanlake.html:72]: Fixed.

**Review by keizer619 — APPROVED** [review]

**Pull request merged**